cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -104,16 +103,18 @@ func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 		return err
 	}
 
+	if c.KeyRef == "" && (c.CertIdentity == "" || c.CertOIDCIssuer == "") {
```

Comment:
Can you add a test for failure if these aren't set?

Reply:
Done.